### PR TITLE
Bump elixir and dependencies versions

### PR DESCRIPTION
**Pull request by killme2008**
This PR bump the elixir to 1.3.0 and all dependencies to latest version.

It seems all the tests are fine:

```
Finished in 0.7 seconds
86 tests, 0 failures
```

And fixed some compile warnings with elixir 1.3.0.

Thanks for your great work.

### CODE REVIEW

File: lib/web/helper.ex [hunk 1]
```diff
@@ -4,7 +4,6 @@ defmodule Beaker.Web.Helper do
   @colors ~w(yellow orange red magenta violet blue cyan green)
 
   def random_color do
-    :random.seed(:os.timestamp)
```
Comment (hahuang65) [inline lib/web/helper.ex:7]: Is there a new way to do this?

**Review by hahuang65 — COMMENTED** [review]
I think the `.travis.yml` file needs to be updated too.

Thank you so much for your work, really appreciate it!

File: lib/web/helper.ex (hunk 1 above)
Comment (killme2008) [reply lib/web/helper.ex:7]: As the rand module describes  http://erlang.org/doc/man/rand.html: 

```
The following algorithms are provided:

exsplus
Xorshift116+, 58 bits precision and period of 2^116-1

exs64
Xorshift64*, 64 bits precision and a period of 2^64-1

exs1024
Xorshift1024*, 64 bits precision and a period of 2^1024-1
The default algorithm is exsplus.If a specific algorithm is required, ensure to always use seed/1 to initialize the state
```
I think the precision is enough here.

**Comment by killme2008** [comment]
@hahuang65 

I updated `.travis.yml`, and the travis test passes. Thanks for your review.

**Pull request merged**